.circleci/config.yml

```diff
@@ -252,6 +253,17 @@ workflows:
           filters:
             tags:
               only: /^v.*/
+      - architect/push-to-app-collection:
+          name: push-to-aws-app-collection
+          app_name: "aws-operator"
+          app_collection_repo: "aws-app-collection"
```

Comment:
Don't we need app_catalog somewhere here?

Reply:
default value is `control-plane-catalog`